modules/tinymce/src/themes/silver/main/ts/modes/Iframe.ts

```diff
@@ -53,13 +54,20 @@ const setupEvents = (editor: Editor) => {
     }
   };
 
-  const scroll = (e) => Events.fireScrollContent(editor, e);
+  const scroll = (e: EventUtilsEvent<Event>) => Events.fireScrollContent(editor, e);
 
   DOM.bind(contentWindow, 'resize', resizeWindow);
   DOM.bind(contentWindow, 'scroll', scroll);
 
   // Bind to async load events and trigger a content resize event if the size has changed
-  const elementLoad = DomEvent.capture(SugarElement.fromDom(editor.getBody()), 'load', resizeDocument);
+  const elementLoad = DomEvent.capture(SugarElement.fromDom(editor.getBody()), 'load', (e) => resizeDocument(e.raw));
```

Comment:
This doesn't compile. The `resizeDocument` function (optionally) passes its argument through to `editor.fire('ResizeContent', e)`, which requires a UI event (as per the docs).

- [link](https://www.tiny.cloud/docs/advanced/events/#editorcoreevents) to where we say what type of payload the `ResizeContent` event has
- [link](https://developer.mozilla.org/en-US/docs/Web/API/Window/resize_event) to where MDN says that event type is a `UIEvent`

Tragically, the event we're getting from `load` to pass through is just an `Event`, as per [this link](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event). Open to suggestions on how to proceed here.